lib/charms/mongodb/v1/shards_interface.py

```diff
@@ -325,6 +354,44 @@ def get_related_shards(self) -> List[str]:
         """Returns a list of related shards."""
         return [rel.app.name for rel in self.charm.model.relations[self.relation_name]]
 
+    def get_unreachable_shards(self) -> List[str]:
```

Comment:
about `-> List[str]`, I don't see a `return unreachable_hosts` in this function.
also sometimes we return a bool instead of List

Reply:
oh gosh you are totally correct 🤦